reference/v8.json

```diff
@@ -2213,6 +2213,61 @@
           "js": "0.21.0"
         }
       }
+    },
+    "circle-stroke-width": {
+      "type": "number",
+      "default": 0,
+      "minimum": 0,
+      "function": "interpolated",
+      "zoom-function": true,
+      "property-function": true,
+      "transition": true,
+      "units": "pixels",
+      "doc": "The stroke width of the circle. Strokes are placed outside of the existing circle radius.",
+      "sdk-support": {
+        "basic functionality": {
+          "js": "master"
+        },
+        "data-driven styling": {
+          "js": "master"
+        }
+      }
+    },
+    "circle-stroke-color": {
+      "type": "color",
+      "default": "#000000",
+      "doc": "The stroke color of the circle.",
+      "function": "interpolated",
+      "zoom-function": true,
+      "property-function": true,
+      "transition": true,
+      "sdk-support": {
+        "basic functionality": {
+          "js": "master"
+        },
+        "data-driven styling": {
+          "js": "master"
+        }
+      }
+    },
+    "circle-stroke-opacity": {
+      "type": "number",
+      "doc": "The stroke opacity of the circle.",
```

Comment:
This sounds a little funky to me — in my head something like "The opacity of the circle's stroke" might sound better? (but if this sounds fine to other people I am okay with it)